### PR TITLE
Add NodeJS SCL to tfm

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -19,12 +21,13 @@
...
 Name: %scl_name
 Version: 5.0
-Release: 8%{?dist}
+Release: 9%{?dist}
 License: GPLv2+
 Group: Applications/File
 Source0: README
 Source1: LICENSE
 Source2: tfm.attr
+Source3: nodejs-symlink-deps
```
Comment (ehelms) [inline packages/foreman/tfm/tfm.spec]: @ekohl I had to add this in order to fix this bug I found -- https://bugzilla.redhat.com/show_bug.cgi?id=1755675

File: packages/foreman/tfm/tfm.spec [hunk 2]
```diff
@@ -137,6 +143,8 @@ Requires: %{scl_runtime}
 Requires: %{scl_runtime}-assets
 Requires: %{scl_prefix_ror}scldevel
 Requires: %{scl_prefix_ruby}scldevel
+Requires: %{scl_prefix_nodejs}scldevel
+Requires: %{scl_prefix_nodejs}build
```
Comment (ehelms) [inline packages/foreman/tfm/tfm.spec]: There are certain macros that need to be available at build time for nodejs packages, e.g. nodejs_sitelib

File: packages/foreman/tfm/tfm.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/tfm/tfm.spec]: That increases my suspicion that we're the only people building RPMs on top of a NodeJS SCL

Comment (ehelms) [reply packages/foreman/tfm/tfm.spec]: Absolutely

**Pull request merged**